src/aabb.typ

```diff
@@ -0,0 +1,61 @@
+/// Compute an axis aligned bounding box (aabb)
+/// for a list of points.
+///
+/// An aabb dictionary has the following keys:
+///  - lo (vector) Min. bounds vector
```

Comment:
Do we have a formal description of what a `vector` and a `point` type is?

Reply:
Not really. We use `vector` for arrays of numbers and `coordinate` for a cetz coordinate. Point should not be used as a type.

Reply:
why not add one in `vector.typ`?

Reply:
> why not add one in `vector.typ`?

What do you mean?

Reply:
Add a description of the `vector` type there, given most should be created from there?

Reply:
We already use vectors everywhere but on the user API.
I actually think it is obvious that a vector is an array of scalars.